keps/sig-node/1967-size-memory-backed-volumes/README.md

```diff
@@ -141,115 +174,139 @@ potentially inaccurate volume size based on node configuration.
 
 ### Feature Enablement and Rollback
 
-_This section must be completed when targeting alpha to a release._
+#### How can this feature be enabled / disabled in a live cluster?
 
-* **How can this feature be enabled / disabled in a live cluster?**
-  - [x] Feature gate (also fill in values in `kep.yaml`)
-    - Feature gate name: SizeMemoryBackedVolumes
-    - Components depending on the feature gate: kubelet
-    - Will enabling / disabling the feature require downtime or reprovisioning
-      of a node? No
+- [x] Feature gate (also fill in values in `kep.yaml`)
+  - Feature gate name: SizeMemoryBackedVolumes
+  - Components depending on the feature gate: kubelet
+  - Will enabling / disabling the feature require downtime or reprovisioning
+    of a node? No
+
+#### Does enabling the feature change any default behavior?
 
-* **Does enabling the feature change any default behavior?**
 Yes, the kubelet will size the empty dir volume to match the precise
 amount of memory the pod is able to write rather than over or undersizing.
 Prior behavior is node dependent, and so pod authors had no mechanism
 to control this behavior properly.
 
-* **Can the feature be disabled once it has been enabled (i.e. can we roll back
-  the enablement)?** Yes
+#### Can the feature be disabled once it has been enabled (i.e. can we roll back the enablement?
+
+Yes
+
+#### What happens if we reenable the feature if it was previously rolled back?
 
-* **What happens if we reenable the feature if it was previously rolled back?**
 Pods that run on that node will have memory backed volumes sized based on Linux
 host default.  The sizing may not align with actual available memory for an app.
 
-* **Are there any tests for feature enablement/disablement?**
+#### Are there any tests for feature enablement/disablement?
+
 No, testing behavior with the feature disabled is dependent on node operating
 system configuration.  The point of this KEP is to address that coupling.
 
 ### Rollout, Upgrade and Rollback Planning
 
-* **How can a rollout fail? Can it impact already running workloads?**
+#### How can a rollout fail? Can it impact already running workloads?
+
 If a pod has more allocatable memory than the default node instance behavior
 of taking 50% node instance memory for sizing emptyDir, a pod could potentially
 write more content to the empty dir volume than previously.  This should have
 no impact on rollout of the cluster or workload.  In practice, applications
 that did exhaust the size of the memory backed volume were not portable across
 instance types or would have had to handle running out of room in that volume.
 
-* **What specific metrics should inform a rollback?**
+#### What specific metrics should inform a rollback?
+
 None.
 
-* **Were upgrade and rollback tested? Was the upgrade->downgrade->upgrade path tested?**
+#### Were upgrade and rollback tested? Was the upgrade->downgrade->upgrade path tested?
+
 I do not believe this is applicable.
```

Comment:
Manual testing is sufficient for this functionality with a cluster which has the feature turned on and then turning it off (or downgrade). I'd hope such tests were performed to ensure the correctness of this functionality, no?

Reply:
I am not sure if they were performed. This was the answer for the PRR questions around 1.22.

I don't know if the upgrade/rollback is really that relevant given that this has been on since 1.22.

Reply:
There are examples across different KEPs, using kind:
https://github.com/kubernetes/enhancements/tree/master/keps/sig-network/1669-proxy-terminating-endpoints#were-upgrade-and-rollback-tested-was-the-upgrade-downgrade-upgrade-path-tested

Where you can disable a feature, and thus achieving a "fake" rollback to test it w/o going this far back. It shouldn't take more than an hour, and would definitely be good to go through to ensure it's good enough for GA.

Reply:
Yeah, it's definitely better than nothing.

Reply:
I pushed an update.